playbooks/cloud-provider-openstack-acceptance-test-csi-cinder/run.yaml

```diff
@@ -117,45 +118,45 @@
           } > /dev/null 2>&1
 
           # Enable services
-          "$kubectl" create -f manifests/cinder-csi-plugin
+          '{{ kubectl }}' create -f manifests/cinder-csi-plugin
           sleep 5
           # If services up
           if timeout 300 bash -c '
               while :
               do
-                  "$kubectl" get pod | sed "1d" | awk '\''$2 != "2/2" || $3 != "Running" {err = 1} END {exit err}'\'' && break
+                  {{ kubectl }} get pod | sed "1d" | awk '\''$2 != "2/2" || $3 != "Running" {err = 1} END {exit err}'\'' && break
                   sleep 1
               done
               '
           then
               echo 'Run services successful'
-              "$kubectl" get pod
+              '{{ kubectl }}' get pod
           else
               echo 'Run services failed'
-              "$kubectl" get pod
+              '{{ kubectl }}' get pod
               exit 1
           fi
 
           # Make test
-          "$kubectl" create -f examples/cinder-csi-plugin/nginx.yaml
+          '{{ kubectl }}' create -f examples/cinder-csi-plugin/nginx.yaml
           # If test passed
           if timeout 300 bash -c '
               while :
               do
-                  [[ $("$kubectl" describe pod nginx | awk "/^Status:/ {print \$2}") == Running ]] && break
+                  [[ $({{ kubectl }} describe pod nginx | awk "/^Status:/ {print \$2}") == Running ]] && break
```

Comment:
{{ kubectl }} should be '{{ kubectl }}'

Reply:
and no '$'

Reply:
Stripped the single quotes is intentional, because the whole while loop has been quoted